examples/java/spring-boot/build.gradle

```
@@ -15,8 +15,15 @@ dependencies {
   implementation 'org.springframework.boot:spring-boot-starter-log4j2'
   implementation 'org.springframework.boot:spring-boot-starter-web'
   testImplementation 'org.springframework.boot:spring-boot-starter-test'
+  testImplementation 'io.rest-assured:json-schema-validator:4.4.0'
 }
 
+task copySchemas(type: Copy) {
```

I don't think we need to use fileTree for this copy task, this might be more readable?
```suggestion
task copySchemas(type: Copy) {
  from '../../../schemas/v1alpha'
  include '*.json'
  into "${project.buildDir}/resources/test/schemas/v1alpha"
}
```

---

Very good spot :+1: